index.js

```diff
@@ -31,21 +39,19 @@ function mnemonicToEntropy (mnemonic, wordlist) {
   wordlist = wordlist || DEFAULT_WORDLIST
 
   var words = unorm.nfkd(mnemonic).split(' ')
-  if (words.length % 3 !== 0) throw new Error('Invalid mnemonic')
+  if (words.length % 3 !== 0) throw new Error(INVALID_MNEMONIC)
   if (words.some(function (word) {
     return wordlist.indexOf(word) === -1
-  })) throw new Error('Invalid mnemonic')
+  })) throw new Error(INVALID_MNEMONIC)
 
   // convert word indices to 11 bit binary strings
   var bits = words.map(function (word) {
     var index = wordlist.indexOf(word)
     return lpad(index.toString(2), '0', 11)
   }).join('')
 
-  // max entropy is 1024; (1024×8)+((1024×8)÷32) = 8448
-  if (bits.length > 8448) {
-    throw new Error('Invalid mnemonic')
-  }
+  if (bits.length < 128) throw new Error(INVALID_MNEMONIC)
+  if (bits.length > 264) throw new Error(INVALID_MNEMONIC)
```

Comment:
@rubensayshi I haven't got time to check, but why does a 24-word mnemonic decode to 264 bits of ENT?
Per the BIP39 vectors...